dev-lang/crystal/crystal-9999.ebuild

```diff
@@ -1,24 +1,35 @@
-# Copyright 1999-2015 Gentoo Foundation
+# Copyright 1999-2016 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 # $Id$
 
-EAPI=5
+EAPI=6
 
 inherit multilib
 
-BV=${PV}-1
+MY_BSV=0.17.4
```

Comment:
Because the crystal can't bootstrap itself, I use this binary package for compiling.